i18n/de_DE.csv

```diff
@@ -158,12 +158,12 @@
 "Magento Enterprise Edition version","Magento Enterprise Edition-Version"
 "Quick reference guide","Kurzanleitung"
 "Log in to the ","Melden Sie sich an "
-"Buckaroo Payment Plaza","Buckaroo Payment Plaza"
+"Buckaroo Buckaroo Plaza","Buckaroo Buckaroo Plaza"
```

Comment:
also this one looks nicer as Buckaroo Payment Plaza ig

Reply:
**This should be Buckaroo Plaza:**
"Buckaroo Plaza","Buckaroo Plaza"